monitor/src/main/java/edu/asu/diging/monitor/web/admin/AddRecipientController.java

```diff
@@ -22,10 +25,23 @@
 	@Autowired
 	private INotificationManager manager;
 	
+	@RequestMapping(value="/admin/recipients/show", method=RequestMethod.GET)
+	public String showRecipients(Model model) {
+		List<RecipientForm> recipientList = new ArrayList<>();
```

Comment:
why are you using the form class here? it's for the list of recipients, not for a form, right?